doc/explanations/min-utxo.rst

```diff
@@ -23,11 +23,11 @@ A minUTxOValue amount of ada pays for ``adaOnlyUTxOSize`` bytes of UTxO storage
 
 The min-ada calculation for any UTxO approximates the above formula. This uses the constants,
 
-  ``coinSize = 2``
+  ``coinSize = 0``
```

Comment:
That is a great catch, thank you!! Could you also put a note in this change?  

``coinSize = 0`` **This is an implementation error, and will be changed to the correct value, ``2``, in the next fork. This
	will decrease the minimum ada value by a small percentage**